core/loadpoint.go

```diff
@@ -381,6 +386,8 @@ func (lp *LoadPoint) evChargeStartHandler() {
 
 	// soc update reset
 	lp.socUpdated = time.Time{}
+
+	lp.startTxn()
```

Comment:
Muss beim Start der 5 Aufrufe auf eine Reihenfolge geachtet werden? Ich sehe Log, asynchrone Events und Statusvariablen.

Ich würde generell erwarten, das die Reihenfolge dann in StopHandler() genau andersherum ist, um Raceconditions o.ä. zu vermeiden. Allerdings kommt da stop Txn() auch ganz am Ende.

Mein Gefühl wäre erstmal den Timer stoppen damit der nichts komisches mehr macht. dann alles was local ist und zum Schluss das Push event?

Reply:
Spielt keine Rolle da stop nur den Zählerstand schreibt.